src/main/scala/codacy/checkstyle/DocGenerator.scala

```diff
@@ -187,7 +187,7 @@ object DocGenerator {
 
   private def withRepository[T](block: File => T): T = {
     File.temporaryDirectory("checkstyle") { directory =>
-      s"git clone git://github.com/checkstyle/checkstyle --depth 1 -b checkstyle-${Versions.checkstyleVersion} $directory".!!
+      s"git clone https://github.com/checkstyle/checkstyle -c advice.detachedHead=false --depth 1 -b checkstyle-${Versions.checkstyleVersion} $directory".!!
```

Comment:
What does `-c advice.detachedHead=false` do?

Reply:
Running the command without this flag was getting me in the 'detached HEAD' state and further the script didn't finish running. After adding it, it worked just fine